### DISCUSSION TITLE
Add OpenAI endpoint variable so can set custom endpoint

### DISCUSSION THREAD

**Discussion by meb427** [ItzCrazyKns/Vane#461] [Ideas]
OpenAI API's are now golden standard for multiple providers.

Lately even GCP/Google announced support for the API format.

Currently we can only add an OpenAI key.
I would like to be able to override that with OpenAI endpoint as that would allow me more choice of backend AI's and that way users can pick a lot more choices. Even Grok have OpenAI endpoint : https://api.groq.com/openai/v1 
https://console.groq.com/docs/openai

Even Ollama already have that endpoint support.

**Comment by ItzCrazyKns** [comment]
Custom OpenAI provider can be used from the settings menu in the UI.

**Comment by meb427** [comment]
I want to use that as variable when starting docker.
